backends/vllm/src/models/llama.rs

```diff
@@ -1,17 +1,25 @@
 use candle_core::{DType, Device, Tensor};
+#[cfg(not(feature = "nccl"))]
 use candle_nn::VarBuilder;
+#[cfg(feature = "nccl")]
+use cudarc::nccl::Comm;
 use hf_hub::{api::sync::ApiBuilder, Repo, RepoType};
 use models::{
-    llama::{Config, LlamaConfig, LlamaEosToks},
+    llama::{Config, LlamaEosToks},
     FlashAttentionMetadata, Llama,
 };
-use std::{path::Path, time::Instant};
+use std::path::Path;
+#[cfg(feature = "nccl")]
+use std::rc::Rc;
+#[cfg(not(feature = "nccl"))]
+use std::time::Instant;
```

Comment:
Are the `use`s correct? e.g. the Instant for non nccl

Reply:
Yes, this is correct.